src/content/en/updates/2019/09/chromium-chronicle-6.md

```diff
@@ -0,0 +1,64 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: The Chromium Chronicle, a monthly series geared specifically to Chromium developers — the developers who build the browser.
+
+{# wf_updated_on: 2019-09-17 #}
+{# wf_published_on: 2019-09-24 #}
+{# wf_tags: chromium-chronicle #}
+{# wf_featured_image: /web/updates/images/generic/cr-chron.jpg #}
+{# wf_featured_snippet: Chrome’s issue tracker, Monorail, offers a grid view that allows you to visualize your issues in a Kanban style board. This episode explains how to use the grid mode. #}
+{# wf_blink_components: N/A #}
+
+# The Chromium Chronicle: Monorail’s Grid View! {: .page-title }
+
+**Episode 6:** September 2019
+
+*by Tiffany in San Francisco*
+
+Chrome’s issue tracker, [Monorail][monorail-homepage], offers a grid view
+that allows you to **visualize your issues in a Kanban style board**.
+When you’re viewing a list of issues, you can click the “Grid” button to
+activate grid mode!
+
+<img src="/web/updates/images/2019/09/chrom-chron-6-screenshot1.png">
+
+While on the grid page, you can customize your view to **sort issues by
+almost any field you want**! Status, Priority, NextAction, Milestone,
+Owner, you name it! 
+
+The flexibility of the grid view allows you to **customize it to fit your
+team’s needs**. For example, below we’ve set up the grid view to show all
+pending Q3 Monorail work, sorted by owner and sprint date.
+
+<img src="/web/updates/images/2019/09/chrom-chron-6-screenshot2.png">
+
+If you need more information on each issue, you can view the grid view with
+“Tile” cells instead. And if you want a bird’s eye view of many, many issues,
+you can **view issues in the grid view as counts**. In fact, the grid view even
+supports loading up to 6,000 issues at once.
+
+<img src="/web/updates/images/2019/09/chrom-chron-6-screenshot3.png">
```

Comment:
Compared to CotT, screenshot 2 and 3 are transposed.

Reply:
Thanks for flagging, Pete. This was definitely not intentional. Will fix.

Reply:
Update: fixed.